src/core/Mixer.cpp

```diff
@@ -144,17 +144,21 @@ void MixerChannel::doProcessing()
 				}
 				else if( volBuf && sendBuf ) // both volume and send have sample-exact data
 				{
-					MixHelpers::addSanitizedMultipliedByBuffers( m_buffer, ch_buf, volBuf, sendBuf, fpp );
+					MixHelpers::addSanitizedMultipliedByBuffers(m_buffer, ch_buf,
```

Comment:
@Veratil it seems this cast is where the build fails.

Reply:
Here's the error:
```
error: invalid static_cast from type 'std::vector<float>*' to type 'float*'
```
To get the internal array use `vector::data` (see [reference](https://en.cppreference.com/w/cpp/container/vector/data))

Reply:
Thanks for saving my sanity. This is why I love working on LMMS. Perfect for learning what you don't know.

Reply:
Now what's going on with the clamp again?  Can you help with this too?

Reply:
Should i cast the 0 too?

Reply:
I have decided to remove the clamp.